reference/test/matrix/ell_kernels.cpp

```diff
@@ -323,4 +324,28 @@ TEST_F(Ell, MovesWithStrideToDense)
 }
 
 
+TEST_F(Ell, ConvertsToCsr)
+{
+    auto csr_mtx = gko::matrix::Csr<>::create(mtx1->get_executor());
+    auto dense_ref = gko::matrix::Dense<>::create(mtx1->get_executor());
+
+    mtx1->convert_to(csr_mtx.get());
+    csr_mtx->convert_to(dense_ref.get());
+
+    GKO_ASSERT_MTX_NEAR(mtx1, dense_ref, 0.0);
+}
+
+
+TEST_F(Ell, ConvertsWithStrideToCsr)
+{
+    auto csr_mtx = gko::matrix::Csr<>::create(mtx2->get_executor());
+    auto dense_ref = gko::matrix::Dense<>::create(mtx2->get_executor());
+
+    mtx2->convert_to(csr_mtx.get());
+    csr_mtx->convert_to(dense_ref.get());
+
+    GKO_ASSERT_MTX_NEAR(mtx1, dense_ref, 0.0);
```

Comment:
Same here to both added tests: You also need to check the result manually if it works the way you want.